nibabel/nifti2.py

```diff
@@ -221,6 +221,17 @@ def _chk_eol_check(hdr, fix=False):
             rep.fix_msg = 'setting EOL check to 13, 10, 26, 10'
         return hdr, rep
 
+    @classmethod
+    def may_contain_header(klass, binaryblock):
```

Comment:
These actually aren't the same test with different numbers, so reusing won't work. Nifti1 returns `hdr_struct['magic'] in (b'ni1', b'n+1')`.